src/Sentry/Integrations/NetFxInstallationsIntegration.cs

```diff
@@ -7,7 +7,7 @@ internal class NetFxInstallationsIntegration : ISdkIntegration
     {
         public void Register(IHub hub, SentryOptions options)
         {
-            if (!Runtime.Current.IsMono())
+            if (!Runtime.Current.IsMono() && !FrameworkInfo.RegistryLocked)
```

Comment:
Do we need `!FrameworkInfo.RegistryLocked`? Can we not just have a try/catch inside `FrameworkInfo.GetLatest()` and be done with it?

Reply:
I added this check to avoid adding an EventProcessor that wasn't going to work due to lack of Registry access. But I could remove it.

Reply:
I think it's fine to keep the event processor, just make it not do anything if it can't access registry.

@bruno-garcia thoughts on this?

Reply:
So the flow that I made was to not throw/register any Exception while creating the Runtime in NET461, but to log it once NetFxInstallationsEventProcessor is Processed.